### ISSUE TITLE
Training settings for the model uploaded on RobustBench

### ISSUE THREAD

**Issue by SravantiAddepalli** [microsoft/robust-models-transfer#13]
Hi authors,

I noticed that the results on [RobustBench](https://robustbench.github.io/#:~:text=NeurIPS%202020-,2,-Do%20Adversarially%20Robust) were different for the models on the Robustness library (62.56/29.22) and for this work (64.02/34.96) for the same architecture ResNet-50, although the same code base was used for the pretraining of both on ImageNet. Could you please share the differences in the training settings of both?
I believe the defaults for the training of models on the Robustness library are the following with 7 attack steps:
`    datasets.ImageNet: {
        "epochs": 200,
        "batch_size":256,
        "weight_decay":1e-4,
        "step_lr": 50
    },`

**Comment by Hadisalman** [comment]
Hi @SravantiAddepalli!

We haven't benchmarked the robustness of our models in this repo cause that wasn't the goal of the paper. Though here are the hyperparams that we used to train these robust models (Also in Appendix A.1 of the paper!):

* batch_size: 512
* momentum: 0.9
* weight_decay: 1e-4
* Epochs: 90
* SGD with step learning rate schedule that starts at lr=0.1 and drops by a factor of 10 every 30 epochs.
* 3 PGD steps
* PGD lr = \eps * 2/3

Hope this helps!

**Comment by SravantiAddepalli** [comment]
Thank you @Hadisalman for the details!
